pom.xml

```diff
@@ -122,7 +122,7 @@
     <nuxeo.features.version>9.10-HF20-SNAPSHOT</nuxeo.features.version>
     <nuxeo.addons.core.version>9.10-HF20-SNAPSHOT</nuxeo.addons.core.version>
 
-    <nuxeo.connect.client.version>1.6.2</nuxeo.connect.client.version>
+    <nuxeo.connect.client.version>1.6.3-SNAPSHOT</nuxeo.connect.client.version>
```

Comment:
Add to `<requireReleaseDeps>` exclusions too

Reply:
Yes please